docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,348 @@ aliases:
+`/v2/session/auth` endpoint on Brigade's API server address, e.g. the hostname
+reserved when setting up ingress.  You'll then use the GitHub OAuth Apps's
+client ID and a generated client secret in the `brigade2-values.yaml` file.
+
+  1. Follow the [GitHub OAuth App] creation instructions, supplying your choice
+    of values for `Application Name`, `Homepage URL` and `Application description`.
+  1. For the `Authorization callback URL`, you'll supply the a value based
+    on the DNS hostname and path mentioned above.  For example, if the DNS
+    hostname is `mybrigade.example.com`, the value would be:
+      ```
+      https://mybrigade.example.com/v2/session/auth
+      ```
+  1. Click 'Register application'
+  1. On the App settings page, there is now a Client ID string. You'll use this
+    value soon.
+  1. Under `Client secrets`, click `Generate a new client secret`
+  1. Save this client secret value now, as it won't be displayed again
+  1. Click 'Update application'
+
+Now that you have values for the GitHub OAuth App's client ID and client
+secret, you're ready to update our Brigade chart values file with this auth
+configuration.
+
+All of it goes under the `apiserver.thirdPartyAuth` section, which we show
+here, with other nearby sections omitted:
+
+```yaml
+apiserver:
+  ## Options for authenticating via a third-party authentication provider.
+  thirdPartyAuth:
+    ## Here we enter our chosen strategy of github
+    strategy: github
+    ## Here we inject the values from the GitHub OAuth App
+    github:
+      ## The client ID goes here
+      clientID: foo
+      ## The client secret goes here
+      clientSecret: foo
+      ## If only users from specific GitHub organizations should be allowed
+      ## to authenticate, list them here.  Otherwise, users from any GitHub
+      ## organization may attempt to authenticate.
+      allowedOrganizations:
+    ## User Session TTL dictates the default time-to-live for user sessions.
+    ## Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+    ## For example, "60s", "2h45m", "168h" (1 week)
+    userSessionTTL: 168h
+    ## If there are users that should be granted admin privileges in Brigade
+    ## from the moment they authenticate, they should be listed here.  For
+    ## instance, the operator doing the deployment and/or the user who will
+    ## be in charge of assigning authorization permissions to authenticated
+    ## users.
+    ## Otherwise, permissions will need to be manually configured for each
+    ## authenticated user.
+    admins:
+      - <enter your name here>
+```
+
+Don't forget to add your GitHub username under the `admins` list, so that you
+as the operator have full admin privileges upon first login to Brigade.  Note:
+since these privileges are granted on first login, adding/revoking these
+permissions must be done via the brig CLI directly rather than re-deploying
+with differing configuration.
```

Comment:
This section could probably be more clear.

Suggestion:

```
Note:
since these privileges are granted on first login, adding/revoking these
permissions for users who have already logged in for the first time must be done via the brig CLI directly rather than re-deploying
with differing configuration.
```